janitor/functions.py

```diff
@@ -3003,27 +2997,64 @@ def find_replace(
 
         # Functional usage
         df = find_replace(
-            df, 'order', {'ice coffee': 'latte', 'regular coffee': 'latte'},
-            match='exact'
+            df,
+            match='exact',
+            order={'ice coffee': 'latte', 'regular coffee': 'latte'},
         )
 
         # Method chaining usage
         df = df.find_replace(
-            'order', {'ice coffee': 'latte', 'regular coffee': 'latte'},
             match='exact'
+            order={'ice coffee': 'latte', 'regular coffee': 'latte'},
         )
 
-
     Example 2: Regular-expression-based matching
 
     .. code-block:: python
 
         # Functional usage
-        df = find_replace(df, 'order', {'coffee$': 'latte'}, match='regex')
+        df = find_replace(
+            df,
+            match='regex',
+            order={'coffee$': 'latte'},
+        )
 
         # Method chaining usage
-        df = df.find_replace('order', {'coffee$': 'latte'}, match='regex')
+        df = df.find_replace(
+            match='regex',
+            order={'coffee$': 'latte'},
+        )
+
+    To perform a find and replace on the entire dataframe,
+    pandas' ``df.replace()`` function provides the appropriate functionality.
+    You can find more detail on the replace_ docs.
+
+    .. _replace: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.replace.html  # noqa: E501
```

Comment:
Do you want to move `# noqa: E501` to the end of the docstring (line 3040) like #593

Reply:
Yes, good catch. I have been sloppy about this. This is even I have to have my code reviewed :smile: (I probably sound like a broken tape recorder by now.) Let me fix it.